vendor/osxlockdown/commands.yaml

```diff
@@ -0,0 +1,276 @@
+  fix_command: |
+    sudo /System/Library/CoreServices/RemoteManagement/ARDAgent.app/Contents/Resources/kickstart -deactivate -stop
+  enabled: true
+
+
+- title: "Enable FileVault"
+  check_command: |
+    diskutil cs list | grep -i "Encryption Status:       Unlocked"
+  comment: "TODO: FileVault can be enabled with `fdesetup enable`, but you need to enter user creds and write down the recovery key. How to do this programmatically, or at least tell the user the steps?  See https://derflounder.wordpress.com/2013/10/22/managing-mavericks-filevault-2-with-fdesetup/"
+  enabled: true
+
+
+- title: "Destroy File Vault Key when going to standby"
+  check_command: |
+    pmset -g | grep DestroyFVKeyOnStandby | grep 1
+  fix_command: |
+    sudo pmset -a destroyfvkeyonstandby 1
+  enabled: true
+
+
+- title: "Enable hibernation mode (no memory power on sleep)"
+  check_command: |
+    pmset -g | grep hibernatemode | grep 25
+  fix_command: |
+    sudo pmset -a hibernatemode 25
+  enabled: true
+
+
+- title: "Enable Gatekeeper"
+  check_command: |
+    spctl --status | grep "assessments enabled"
+  fix_command: |
+    sudo spctl --master-enable
+  enabled: true
+
+
+- title: "Enable Firewall"
+  check_command: |
+    test $(defaults read /Library/Preferences/com.apple.alf globalstate) -ge 1
+  fix_command: |
+    defaults write /Library/Preferences/com.apple.alf globalstate -int 1
+  enabled: true
+
+
+- title: "Enable Firewall Stealth Mode"
+  check_command: |
+    /usr/libexec/ApplicationFirewall/socketfilterfw --getstealthmode | grep "Stealth mode enabled"
+  fix_command: |
+    /usr/libexec/ApplicationFirewall/socketfilterfw --setstealthmode on
+  enabled: true
+
+
+- title: "Disable signed apps from being auto-permitted to listen through firewall"
+  check_command: |
+    defaults read /Library/Preferences/com.apple.alf allowsignedenabled | grep 0
+  fix_command: |
+    defaults write /Library/Preferences/com.apple.alf allowsignedenabled -bool false
+  enabled: true
+
+
+- title: "Disable iCloud drive"
+  check_command: |
+    if [ -n "$(defaults read NSGlobalDomain NSDocumentSaveNewDocumentsToCloud | grep "0")" ]; then exit 0; fi; exit 1;
+  fix_command: |
+    defaults write NSGlobalDomain NSDocumentSaveNewDocumentsToCloud -bool false
+  comment: "TODO: Disable icloud entirely. This checks if a value is not set or set to 0"
+  enabled: true
+
+
+- title: "Require an administrator password to access system-wide preferences"
+  check_command: |
+    if [ -n "$(security authorizationdb read system.preferences 2> /dev/null | grep -A1 shared | grep -E '(true|false)' | grep 'false')" ]; then exit 0; fi; exit 1
+  fix_command: |
+    security authorizationdb read system.preferences > /tmp/system.preferences.plist &&/usr/libexec/PlistBuddy -c "Set :shared false" /tmp/system.preferences.plist && security authorizationdb write system.preferences < /tmp/system.preferences.plist
+  enabled: true
+
+
+- title: "Disable IPv6"
```

Comment:
✂️ , IPv6 routing issues were fixed in [El Capitan v10.11](https://support.apple.com/en-au/HT205267)